hypothesis-python/tests/cover/test_filter_rewriting.py

```diff
@@ -343,14 +344,39 @@ def test_warns_on_suspicious_string_methods(method):
     assert fs.min_size == 1
 
 
-@pytest.mark.parametrize("method", [str.isidentifier, str.isalnum])
+@pytest.mark.parametrize("method", [str.isalnum])
 def test_bumps_min_size_and_filters_for_content_str_methods(method):
     s = unwrap_strategies(st.text())
     fs = s.filter(method)
     assert fs.filtered_strategy.min_size == 1
     assert fs.flat_conditions == (method,)
 
 
+# Should we deterministically check whether ascii or not or st.characters fine?
+@pytest.mark.parametrize("al", [None, "cdef123", "cd12¥¦§©"])
```

Comment:
Fine to ignore the non-`OneCharStringStrategy` case (fyi `sampled_from()` is automatically converted in `text()`!); definitely good to test both ascii and non-ascii.